features/support/object_model_pages/ivl/registration/ivl_personal_information.rb

```diff
@@ -323,4 +323,8 @@ def self.help_me_sign_up_btn
   def self.save_and_exit_link
     '.interaction-click-control-save---exit'
   end
+
+  def self.save_link
+    '.interaction-click-control-save'
```

Comment:
could we use a data-cuke attribute here?

Reply:
I just realized this was not actually used, I'll remove that and push again